src/java/org/apache/cassandra/cql3/QueryProcessor.java

```diff
@@ -516,19 +516,28 @@ public static UntypedResultSet execute(String query, ConsistencyLevel cl, Object
     public static UntypedResultSet execute(String query, ConsistencyLevel cl, QueryState state, Object... values)
     throws RequestExecutionException
     {
-        try
```

Comment:
I have removed the try-catch block because the only caller of this method is `SystemDistributedKeyspace#viewStatus`, who simply swallows the exception making this wrapping of `RequestValidationException` no-op.

Reply:
I don't know, how do you think this could fail?

Reply:
Yes, the generic try-catch block in that variant of execute looks like a reminder of the original implementation. The method is only used by `SystemDistributedKeyspace#viewStatus`, which shallows the exception because it's only interested in collecting view statuses for a nodetool command. So it's a no-op. Now we want to re-use the method for `QueryTester`, but the useless exception replacement gets in the way and hides the original exception, which we want for our test checks. 

Do you agree on simply removing the try-catch block, since it doesn't seem to use any purpose? Or I can add a variant of the `QueryProcessor#execute` method without the exception replacement, or even embed it in `CQLTester#executeFormattedQuery`, given how simple it is.